fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -355,15 +396,18 @@ private fun TranslationsDropdown(
 
         Spacer(modifier = Modifier.height(4.dp))
 
+        var initialValue = stringResource(R.string.translations_bottom_sheet_default_dropdown_selection)
+        if (initiallySelected?.localizedDisplayName != null) {
+            initialValue = initiallySelected.localizedDisplayName!!
+        }
```

Comment:
You can get around the compiler's inability to avoid a force unwrap (`!!`) here by using a receiver block instead: 
```suggestion
        initiallySelected?.localizedDisplayName?.let {
            initialValue = it
        }
```

Reply:
Ty, updated